openapi.yaml

```diff
@@ -281,9 +286,6 @@ components:
         data:
           type: object
           properties:
-            id:
-              type: string
```

Comment:
The `EmployeeResource` is used in the `POST` and of course we don't know the ID then so there's no point in requiring it.